portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase

```diff
@@ -4026,6 +4026,95 @@ definition {
 			fieldValue = "Checkbox 1");
 	}
 
+	@priority = "5"
+	test SubmitFormWithRequiredConfirmationNumericField {
+		property portal.acceptance = "true";
+		property portal.upstream = "true";
+		property test.name.skip.portal.instance = "Forms#SubmitFormWithRequiredConfirmationNumericField";
+
+		Navigator.openURL();
+
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName();
+
+		Form.editDescription();
+
+		Form.addNumericField(fieldPositionNumber = "1");
+
+		var fieldNumericField = FormFields.getFieldName();
```

Comment:
@gleicelisbino, how about to rename this variable to `numericFieldName` to better reflect its purpose?